hs_core/templates/pages/genericresource.html

```diff
@@ -1044,6 +1096,180 @@ <h3 style="color: red" id="file-type-error"></h3>
                 $(el).hide().fadeIn(400);
             }
 
+            function showAddEditExtraMetaPopup(edit, row_id_str)
+            {
+                $("#edit_extra_meta_row_id").val('');
+                $("#old_extra_meta_name").val('');
+                $("#extra_meta_name_input").val('');
+                $("#extra_meta_value_input").val('');
+                if (edit)
+                {
```

Comment:
I know it is a formatting preference, but to comply with the rest of code formats, I'd suggest to move ```{``` to the line above to save some extra lines.